AsBuiltReport.Fortinet.FortiGate.json

```diff
@@ -11,7 +11,8 @@
     "Options": {
         "Port": 443,
         "VDOM": "",
-        "PolicyLayout": "all"
+        "PolicyLayout": "all",
+        "ExcludeDownInterfaces": true
```

Comment:
Need a last change, add options on README.md and look good

Reply:
forget to push change ?

Reply:
try now